source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -103,6 +103,11 @@ The Wazuh agent is a single and lightweight monitoring software. It is a multi-p
 
                         # pacman --noconfirm -Syu curl gcc make sudo wget expect gnupg perl-base perl fakeroot python brotli automake autoconf libtool gawk libsigsegv nodejs base-devel inetutils cmake
 
+                .. group-tab:: APK
+
```

Comment:
It would be useful to include here a brief introduction explaining that APK is the `Alpine Package Keeper`, the default package manager for Alpine Linux.